hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/replicatedmap/ReplicatedMapGetMessageTask.java

```diff
@@ -59,7 +60,7 @@ protected Object processResponseBeforeSending(Object response) {
         ReplicatedMapService replicatedMapService = getService(ReplicatedMapService.SERVICE_NAME);
         if (replicatedMapService.getReplicatedMapConfig(parameters.name).isStatisticsEnabled()) {
             LocalReplicatedMapStatsImpl stats = replicatedMapService.getLocalReplicatedMapStatsImpl(parameters.name);
-            stats.incrementGets(System.nanoTime() - startTimeNanos);
+            stats.incrementGets(TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startTimeNanos));
```

Comment:
I think we should rather report it as nanos and change the unit to `NS` too, similarly to the map counterpart: https://github.com/hazelcast/hazelcast/blob/97ff57ba92dc1fbfa77720885e7db40757999080/hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/MapGetMessageTask.java#L71 

I'd expect with reporting as millis, we typically report 0.

The same fields in the map are not metrics though :thinking: This seems to be an error, I send a PR to fix this.
https://github.com/hazelcast/hazelcast/blob/296aac5c8a3a4970851afc55c9e91622db75c131/hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/LocalMapStatsImpl.java#L140

Reply:
After taking another look, we do have metrics for maps' total get latency, but we do it a bit different:
https://github.com/hazelcast/hazelcast/blob/296aac5c8a3a4970851afc55c9e91622db75c131/hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/LocalMapStatsImpl.java#L311-L315

So we accumulate nanos, but convert it to millis for the metrics collector. I believe this is the right way to do.
@alex-dukhno @puzpuzpuz :point_up:

Reply:
> I believe this is the right way to do.

I agree. We should do the same here.

Reply:
@blazember how about `MAX_*_LATENCY`? they are collected in the same methods as `total*Latencies`
e.g. https://github.com/hazelcast/hazelcast/blob/296aac5c8a3a4970851afc55c9e91622db75c131/hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/LocalReplicatedMapStatsImpl.java#L238-L242

Reply:
I think `MS` is good, the max is very likely to be in the `MS` range anyways. Also, the map stats report it as `MS`, the best is to be consistent across the structures.